newIDE/app/src/LayersList/index.js

```diff
@@ -178,6 +211,14 @@ export default class LayersList extends Component<Props, State> {
           </FullSizeMeasurer>
           <Column>
             <Line justifyContent="flex-end" expand>
+              <RaisedButton
+                label={<Trans>Add lighting layer</Trans>}
+                primary
```

Comment:
Let's not make this button primary, and let's show it on a line after Add a layer - to ensure first time users are drove to use "Add a layer" first.